tsconfig.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
This file used to be generated: https://github.com/eclipse-theia/theia/pull/14481/files#diff-21d974bacb2fdab12e4e74074940b9e74d897b70d163686bfbe5a7048f62d056. If we revive it, we need to revive the generation code, as well. However, I'm not sure treating the monorepo as a compilable artifact is proper. Is there another way we can configure what needs to be documented? How are the examples being filtered?

Reply:
Because we're no longer using it to compile the whole monorepo.